openml/tasks/task.py

```diff
@@ -80,53 +127,6 @@ def get_X_and_y(self):
         X_and_y = dataset.get_data(target=self.target_name)
         return X_and_y
 
-    def get_train_test_split_indices(self, fold=0, repeat=0, sample=0):
```

Comment:
sorry, can't comment in things outside the diff, but above is another reference to the task type ids